src/Parameters.hs

```diff
@@ -0,0 +1,31 @@
+module Parameters (
+    Parameters(..),
+    argParser,
+    paramInfo
+) where
+
+import Options.Applicative
```

Comment:
Need explicit imports here

Reply:
We are missing `-Wmissing-import-lists` in `cabal` file

Reply:
Yep, lets get that in there